admin/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
remove LICENSE file? we have another one in root.

Reply:
Removed